python/tvm/relay/frontend/onnx.py

```diff
@@ -478,7 +478,7 @@ def _impl_v1(cls, inputs, attr, params):
                 attr["dilations"] = [1] + list(attr["dilations"])
             if "pads" in attr:
                 attr["pads"] = [0, attr["pads"][0], 0, attr["pads"][1]]
-
+        attr["channels"] = kernel_shapes[0][0]
```

Comment:
does this change relate to the subject of the PR or it is side fix? Don't see where "channels" attribute is used in other changes of this PR

Reply:
This is a side fix, I utilized it for an intermediate solution for conv, but I guess I don't technically need it. It doesn't seem to be required in many of the relay passes.